### PR TITLE
daw

### PR DESCRIPTION
**Pull request by BillaAhmed3333334566**
Hello dude

### CODE REVIEW

**Review by BilalAhmed23343 — COMMENTED** [review]
n00b

**Comment by BillaAhmed3333334566** [comment]
why ?

File: style.css [hunk 1]
```diff
@@ -3,6 +3,7 @@ div.main-body {
   grid-template-columns: minmax(300px, auto) 1fr;
   grid-template-areas: "navbar mainContent";
   grid-gap: 20px;
+  background-color: black;
```
Comment (BilalAhmed23343) [inline style.css:6]: hey whats this bollsheet

File: style.css [hunk 2]
```diff
@@ -3,6 +3,7 @@ div.main-body {
...
 }
```
Comment (BilalAhmed23343) [inline style.css:7]: dwa

**Pull request merged**